diff --git a/cert.go b/cert.go
index 4ce36cc..dd8dd85 100644
--- a/cert.go
+++ b/cert.g
@@ -59,7 +59,7 @@ func (m *mkcert) makeCert(hosts []string) {
        // Certificates last for 2 years and 3 months, which is always less than
        // 825 days, the limit that macOS/iOS apply to all certificates,
        // including custom roots. See https://support.apple.com/en-us/HT210176.
-       expiration := time.Now().AddDate(2, 3, 0)
+       expiration := time.Now().AddDate(0, 0, 397)

        tpl := &x509.Certificate{
                SerialNumber: randomSerialNumber(),
